goodkey/weak.go

```diff
@@ -0,0 +1,59 @@
+package goodkey
```

Comment:
Was this intended to be included in this PR or did it sneak in? I don't see it referenced anywhere in the load generator code.